eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Cool, Thanks Chitin, hope you have tested this well, since its eagle-external anyway, I will merge this once the ci passes.